account/account.go

```diff
@@ -66,8 +66,7 @@ func (Aggregate) RouteCommandToInstance(m dogma.Message) string {
 	}
 }
 
-// HandleCommand handles a command message that has been routed to this
-// handler.
+// HandleCommand handles a command message that has been routed to this // handler.
```

Comment:
Comment slashes midway through sentence.

Reply:
Ok not sure what happened there. Will fix.